src/components/modules/IconsHome/SearchAddTopSection/index.tsx

```diff
@@ -27,7 +28,8 @@ export const SearchAddTopSection: FC<Props> = ({ searchQuery, setSearchQuery })
       <GlobalHotKeys keyMap={keyMap} handlers={handlers} />
       <Search
         name="icons-search"
-        placeholder="Quick search for icons (⌘f)"
+        // eslint-disable-next-line prefer-template
+        placeholder={'Quick search for icons (' + getSearchShortcut() + ')'}
```

Comment:
Lets use JS template strings. Its much more readable and preferred one these days.
```js
placeholder={`Quick search for icons (${getSearchShortcut()})`}
```